src/schema/v2/artwork/collectorSignals.ts

```diff
@@ -61,20 +106,50 @@ const collectorSignalsLoader = async (
 
   // Handle signals for auction artworks
   if (isInSale && auctionsCollectorSignalsEnabled) {
-    const activeSaleArtwork = await getActiveSaleArtwork(
+    const activeLotData = await getActiveSaleArtwork(
       {
         artworkId,
         saleIds: artwork.sale_ids,
       },
       ctx
     )
 
-    if (activeSaleArtwork) {
+    const activeAuctionLot = activeLotData && !activeLotData.sale.ended_at
```

Comment:
Question: Do we still need to check `ended_at` if we are already fetching from _active_ sale above?

Reply:
Oh I see your [comment below](https://github.com/artsy/metaphysics/pull/5889/files#r1707107448)! 👍